src/components/DpTooltip/utils/tooltip.js

```diff
@@ -59,11 +63,14 @@ const initTooltip = (el, value, options) => {
 
   const id = `tooltip-${uuid()}`
   const zIndex = getZIndex(el)
+  const tooltipData = { id: id, value: value }
+
+  tooltips.push(tooltipData)
```

Comment:
why do You use an array and not an Object?
```suggestion
  tooltips[id] = tooltipData
```
with that you can avoid the loop in line 145

Reply:
refactored it here https://github.com/demos-europe/demosplan-ui/pull/542/commits/f4102553846455c57f7e55a9f4174056425a5cf8. Use an object instead of an array